extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -249,14 +251,19 @@ private static ParsedPersistenceXmlDescriptor generateReactivePersistenceUnit(
         }
 
         // Query
-        if (persistenceUnitConfig.batchFetchSize > 0) {
+        int batchSize = firstPresent(persistenceUnitConfig.fetch.batchSize, persistenceUnitConfig.batchFetchSize)
+                .orElse(16);
```

Comment:
just a suggestion.
what do you think this value "16" change for constant `DEFAULT BATCH_SIZE`?

Motivation:
Just to be easy read coding

Reply:
While I agree it's good practice in general, here I'm not sure:

1. This code is all about setting defaults, so it feels wrong to move the default away from the code.
2. There are references to this default value (`16`) elsewhere, in particular in an annotation on `io.quarkus.hibernate.orm.deployment.HibernateOrmConfigPersistenceUnit.HibernateOrmConfigPersistenceUnitFetch#batchSize`, and that reference cannot use the same constant (because it's a string). Moving the value to a constant would create a false sense of security, as it would seem that you only need to change that to update the default, while in fatch it's not enough.
3. All the code below and above this line is setting defaults as well, and none of it uses constants for some reason. I think consistency would be best?

Reply:
the use of `orElse` makes it very clear as well.
This case doesn't really look like a `magic number`.
it was just something for us to think and a little vanity :smile: 

Thank you for your attention.